src/three-components/TextureUtils.ts

```diff
@@ -366,39 +373,61 @@ void main() {
     });
 
     blurScene.add(new Mesh(geometry, blurMaterial));
+    const blurUniforms = blurMaterial.uniforms;
+
+    const cubeTexture = cubeTarget.texture;
+    const params = {
+      type: cubeTexture.type,
+      format: cubeTexture.format,
+      encoding: cubeTexture.encoding,
+      generateMipmaps: cubeTexture.generateMipmaps,
+      minFilter: cubeTexture.minFilter,
+      magFilter: cubeTexture.magFilter
+    };
 
-    const blurCamera = new CubeCamera(0.1, 100, cubeResolution);
-    const tempTarget = blurCamera.renderTarget;
-    tempTarget.texture.type = UnsignedByteType;
-    tempTarget.texture.format = RGBEFormat;
-    tempTarget.texture.encoding = RGBEEncoding;
-    tempTarget.texture.magFilter = NearestFilter;
-    tempTarget.texture.minFilter = NearestFilter;
-    tempTarget.texture.generateMipmaps = false;
-
-    const gammaOutput = this.renderer.gammaOutput;
-    const toneMapping = this.renderer.toneMapping;
-    const toneMappingExposure = this.renderer.toneMappingExposure;
-
-    this.renderer.toneMapping = LinearToneMapping;
-    this.renderer.toneMappingExposure = 1.0;
-    this.renderer.gammaOutput = false;
-
-    blurMaterial.uniforms.latitudinal.value = false;
-    blurMaterial.uniforms.tCube.value = cubeTarget.texture;
-    blurCamera.update(this.renderer, blurScene);
+    // Three.js bug: CubeCamera.d.ts constructor is not up to date with
+    // CubeCamera.js
+    let blurCamera = new (CubeCamera as any)(0.1, 100, cubeResolution, params);
+    const tempTexture = blurCamera.renderTarget.texture;
 
-    blurMaterial.uniforms.latitudinal.value = true;
-    blurMaterial.uniforms.tCube.value = tempTarget.texture;
-    blurCamera.renderTarget = cubeTarget;
+    blurUniforms.latitudinal.value = false;
+    blurUniforms.tCube.value = cubeTexture;
+    blurUniforms.inputEncoding.value = encodings[cubeTexture.encoding];
+    blurUniforms.outputEncoding.value = encodings[tempTexture.encoding];
     blurCamera.update(this.renderer, blurScene);
 
-    this.renderer.toneMapping = toneMapping;
-    this.renderer.toneMappingExposure = toneMappingExposure;
-    this.renderer.gammaOutput = gammaOutput;
+    if (outputEncoding != null && outputEncoding === GammaEncoding &&
```

Comment:
Strictly speaking, you probably don't need the first clause of this condition because `outputEncoding === GammaEncoding` will always be false for every non-number value.